strings/base_implements.h

```diff
@@ -1041,8 +1041,6 @@ namespace winrt::impl
             return Windows::Foundation::TrustLevel::BaseTrust;
         }
 
-        using is_factory = std::disjunction<std::is_same<Windows::Foundation::IActivationFactory, I>...>;
-
```

Comment:
Deleting this public type is a potential breaking change. Do we care?

Reply:
I don't think it matters too much - developers can easily sniff this out themselves - and its not likely to be something a project would depend on, I think.